### PR TITLE
Completely remove remaining use of Vows

### PR DESCRIPTION
**Pull request by brodybits**
In followup to PR #121 (port to Jest) and related to PR #89 (use mostly strict assertions), this completely removes remaining use of Vows and related package scripts no longer needed.

TODO items:

- [ ] resolve known conflicts since I started working on this before merging PR #124
- [ ] I would like to finish review & merge of PR #89 before merging this one.

/cc @awwright @karfau

### CODE REVIEW

**Review by karfau — APPROVED** [review]
:tada: Really nice!
Since the custom assertions don't really rely on `vows` other then in tests, we can actually really land this one!

EDIT: Github only showed me the conflicts after I approved, not sure why. I'm not able to dismiss my own approval.

**Comment by brodybits** [comment]
Hey @karfau can you check these changes once again?

I merged changes from master, using `git checkout --ours test` to avoid some conflicts, and re-applied Prettier formatting to the affected modules.

**Pull request merged**
